### PR TITLE
Fix console deprecation warnings in core code

### PR DESCRIPTION
**Pull request by peterflynn**
Fix some moved APIs that were left over from cleanup PR #10641.

Looks like these got missed -- fyi @le717.  My bad too since I didn't notice this during the code review.

### CODE REVIEW

**Comment by le717** [comment]
:+1:

**Comment by le717** [comment]
Sorry about this mess again. I did a Find in Files search from within Brackets and somehow I must have overlooked these. :(

File: src/editor/EditorStatusBar.js [hunk 1]
```diff
@@ -37,9 +37,8 @@ define(function (require, exports, module) {
         AppInit                      = require("utils/AppInit"),
         DropdownButton               = require("widgets/DropdownButton").DropdownButton,
         EditorManager                = require("editor/EditorManager"),
-        MainViewManager     = require("view/MainViewManager"),
```
Comment (MarcelGerber) [inline src/editor/EditorStatusBar.js]: Just as I saw that change, I think you can remove one level of indentation here.

Comment (peterflynn) [inline src/editor/EditorStatusBar.js]: Makes sense -- could go back two levels, even.  Change pushed.

**Comment by le717** [comment]
@peterflynn Anything preventing this from being merged?

**Comment by peterflynn** [comment]
@le717 It needs to be reviewed by another committer before it can be merged

**Comment by zaggino** [comment]
Reviewed and merged.

**Pull request merged**